libs/designer-ui/src/lib/querybuilder/index.tsx

```diff
@@ -76,8 +86,21 @@ export const QueryBuilderEditor = ({ getTokenPicker, groupProps, readonly, onCha
     setRootProp(newProps);
   };
 
+  let description;
+  if (showDescription) {
```

Comment:
nit: but because we're only rendering it on showDescription, we don't need to optionally set the description